### PR TITLE
LPS-70364 Removes the table-cell CSS class that is not supported by Lexicon from taglibs.

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by sharonchoi** [comment]
Hi @Robert-Frampton! This pull is a release blocker for FP11, so I was wondering if you can prioritize reviewing this when you can. Thank you! :)

File: portal-web/docroot/html/taglib/ui/search_iterator/list.jsp [hunk 1]
```diff
@@ -274,7 +274,7 @@ if (iteratorURL != null) {
 				}
 			%>
 
-				<td class="table-cell <%= columnClassName %>" colspan="<%= entry.getColspan() %>">
+				<td class="table-cell-content <%= columnClassName %>" colspan="<%= entry.getColspan() %>">
```
Comment (Robert-Frampton) [inline portal-web/docroot/html/taglib/ui/search_iterator/list.jsp]: Hey @fernandosouza 

I think we might be able to remove this class. Usually the `table-cell-content` class is used when wanting to make one cell larger than the others in it's column, so adding it to all of them is probably unnecessary. Was there another reason you added it that I'm not aware of?

**Comment by Robert-Frampton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by fernandosouza** [comment]
@Robert-Frampton, branch updated. 😃

**Comment by Robert-Frampton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**